python/cudf/dataframe/datetime.py

```diff
@@ -189,6 +190,24 @@ def default_na_value(self):
             raise TypeError(
                 "datetime column of {} has no NaN value".format(self.dtype))
 
+    def sort_by_values(self, ascending=True, na_position="last"):
```

Comment:
Ideally there should be more commonality between all the column types this might happen during a larger refactor after 0.6 ?

Reply:
Yes